dandiapi/api/tests/test_info.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+
+from dandiapi.api.views.info import schema_url
+import versioneer
+
+
+def test_rest_info(api_client):
+    resp = api_client.get('/api/info/')
+    assert resp.status_code == 200
+    assert resp.json() == {
+        'schema_version': settings.DANDI_SCHEMA_VERSION,
+        'schema_url': schema_url,
+        'version': versioneer.get_version(),
+        'cli-minimal-version': '0.14.2',
+        'cli-bad-versions': [],
+        'services': {
+            'api': {'url': settings.DANDI_API_URL},
+            'webui': {'url': settings.DANDI_WEB_APP_URL},
+            'jupyterhub': {'url': settings.DANDI_JUPYTERHUB_URL},
+        },
+    }
```

Comment:
Since this endpoint is something that the CLI depends on, I think having a similar test as part of the CLI integration tests would be valuable (thoughts @jwodder @yarikoptic?)

Reply:
There effectively is such a test already: https://github.com/dandi/dandi-cli/blob/00ee4ddf8785a91ff5bd893497b018ef5fce6437/dandi/tests/test_utils.py#L303

Reply:
That's a really good point. I think adding a CLI Integration test is important, since that's what would break if we were to change this. We probably want to merge this as soon as possible, but that test could be added after the fact.

Reply:
@jwodder I see a couple of problems with that test

1. We're not specifying the `DANDI_REDIRECTOR_BASE` environment variable currently, so this test is either skipped or tested against the currently deployed prod (not totally sure of the behavior from looking at that file). In either case, the current change isn't tested.
2. Even if we did specify that env variable, this redirection doesn't happen locally, it only happens in Netlify, so the test would fail in that way.

We should probably at least add an almost identical test aimed at the locally spun up API instance, instead of prod.